draco/core/containers.py

```diff
@@ -2300,10 +2300,46 @@ class DelaySpectrum(ContainerBase):
         }
     }
 
+    def __init__(self, weight_boost=1.0, *args, **kwargs):
+        super(DelaySpectrum, self).__init__(*args, **kwargs)
+        self.attrs["weight_boost"] = weight_boost
+
+    @property
+    def spectrum(self):
+        return self.datasets["spectrum"]
+
+    @property
+    def weight_boost(self):
+        return self.attrs["weight_boost"]
+
+
+class DelayTransform(ContainerBase):
+    """Container for a delay spectrum."""
```

Comment:
I think it could be beneficial to elaborate a bit more on the difference between a delay power spectrum and a delay spectrum here, especially since we've used the terms somewhat interchangeably (specifically, why does `DelayTransform` say it holds a delay spectrum and not `DelaySpectrum`?)

Reply:
Agreed, the conventions are confusing. I've added some notes to the `DelaySpectrum` docstring to clarify this.

What I'd really like to do is rename `DelaySpectrum` -> `DelayPowerSpectrum` and `DelayTransform` -> `DelaySpectrum` to make the meanings clearer, but this would break all sorts of other code. A reasonable compromise might be `DelaySpectrum` -> `DelayPowerSpec` and `DelayTransform` -> `DelaySpec`, and then aliasing `DelaySpectrum` to `DelayPowerSpec` for backwards-compatibility.